__tests__/core/validation/ValidationRuleset.spec.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2020 NEM Foundation (https://nem.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { validate } from 'vee-validate'
+import {ValidationRuleset} from '@/core/validation/ValidationRuleset'
+
+describe('test stripTags',()=>{
+
+  test('stripTags should not be used for messages',()=>{
+    validate('<message>', ValidationRuleset.message).then(result=>{
+      expect(result.valid).toBe(false)
+    }) 
+  })
+
+  test('stripTags should not be used for newAccountName',()=>{
+    validate('<newAccountName>', ValidationRuleset.newAccountName).then(result=>{
+      expect(result.valid).toBe(false)
+    })
+  })
+  
+  test('stripTags should not be used for accountWalletName',()=>{
+    validate('<newAccountName>', ValidationRuleset.accountWalletName).then(result=>{
+      expect(result.valid).toBe(false)
+    })
+  })
+})
```

Comment:
these tests are not testing the actual stripTags functionality of .... removing tags.